### PR TITLE
Storage 2016-01-01

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java [hunk 1]
```diff
@@ -7,14 +7,13 @@
...
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
-import com.microsoft.azure.management.storage.implementation.api.UsageInner;
 import com.microsoft.azure.management.storage.implementation.api.UsageName;
 import com.microsoft.azure.management.storage.implementation.api.UsageUnit;
 
 /**
  * An immutable client-side representation of an Azure storage resource usage info object.
  */
-public interface Usage extends Wrapper<UsageInner> {
+public interface Usage extends Wrapper<com.microsoft.azure.management.storage.implementation.api.Usage> {
```
Comment (martinsawicki) [inline azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java]: rename to UsageInner?

Comment (martinsawicki) [inline azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java]: logged as issue, so we can move on with the PR

Comment (jianghaolu) [inline azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java]: It's generated code. Can't rename.

Comment (martinsawicki) [inline azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java]: ok

Comment (jianghaolu) [inline azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/Usage.java]: What's happening is they moved their list into an object like `StorageAccountUsageListResult` and `Usage` is no longer a top level.

**Pull request merged**